src/components/addedGrid/AddedGrid.jsx

```diff
@@ -1,43 +1,44 @@
 "use client";
```

Comment:
I guess it's preferable to use a map here instead

Reply:
Yes, i thought about it, but i kept to the data fetching